app/src/main/java/ca/fuwafuwa/kaku/Ocr/OcrRunnable.java

```diff
@@ -137,26 +140,11 @@ public void stop(){
         }
     }
 
-    private Bitmap getReadyScreenshotBox(BoxParams box, int attempts) throws OutOfMemoryError, StackOverflowError, TimeoutException, FileNotFoundException {
-
-        if (attempts > 10){
-            return null;
-        }
+    private Bitmap getReadyScreenshotBox(BoxParams box) throws OutOfMemoryError, StackOverflowError, TimeoutException, FileNotFoundException {
```

Comment:
I guess we don't really need a recursive function here either, a while loop would be fine as well.